api/capi/include/nnstreamer.h

```diff
@@ -168,6 +168,17 @@ typedef enum _ml_tensor_type_e
   ML_TENSOR_TYPE_UNKNOWN         /**< Unknown type */
 } ml_tensor_type_e;
 
+/**
+ * @brief Possible data format/layout of Tensor in NNStreamer.
+ * @since_tizen 6.0
+ */
+typedef enum _ml_tensor_layout_e
+{
+  ML_TENSOR_LAYOUT_NHWC = 0,      /**< Channel first layout */
+  ML_TENSOR_LAYOUT_NCHW,          /**< Channel last layout */
+  ML_TENSOR_LAYOUT_NONE,          /**< Unknown layout */
```

Comment:
Set ```ML_TENSOR_LAYOUT_NONE = 0``` as it is the default value.
```s/Unknown layout/It has unknown layout or it does not care the data layout./```

Reply:
And be aware, you are modifying the public API :)

There will be a lot of follow-up "paper work"

Reply:
Updated.